Cargo.toml

```diff
@@ -33,16 +33,18 @@ service_account = ["hyper-rustls", "rustls", "rustls-pemfile"]
 [dependencies]
 anyhow = "1.0.38"
 async-trait = "^0.1"
-base64 = "0.21"
+base64 = "0.22"
 futures = "0.3"
-http = "0.2"
-hyper = { version = "0.14", features = ["client", "server", "tcp", "http2"] }
-hyper-rustls = { version = "0.25", optional = true, features = ["http2"] }
-hyper-tls = { version = "0.5.0", optional = true }
+http = "1"
+http-body-util = "0.1"
+hyper = { version = "1", features = ["http2"] }
+hyper-util = "0.1"
+hyper-rustls = { version = "0.27", optional = true, default-features = false, features = ["http2"] }
+hyper-tls = { version = "0.6.0", optional = true }
 itertools = "0.12"
 log = "0.4"
 percent-encoding = "2"
-rustls = { version = "^0.22.1", optional = true }
+rustls = { version = "^0.23", optional = true, features = ["ring"] }
```

Comment:
ideally there'd be a feature to let user decide between aws-lc-rs & ring, but current code relies on ring